src/lib/holds.js

```diff
@@ -26,31 +26,34 @@ function insertEntryByName (accountName, transferId, transaction) {
 
 function holdFunds (transfer, transaction) {
   return Promise.all(transfer.debits.map((debit) => {
-    return Promise.all([
-      adjustBalance(debit.account, -debit.amount, transaction),
-      adjustBalance('hold', debit.amount, transaction),
-      insertEntryByName(debit.account, transfer.id, transaction)
-    ])
+    return adjustBalance(debit.account, -debit.amount, transaction).then(() => {
```

Comment:
No, see the diff, `Promise.all` does them in "parallel" (quotes because javascript is single-threaded)